File: app/src/main/java/mb00/android/codehub/ui/login/inject/LoginActivityComponent.java
package mb00.android.codehub.ui.login.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.login.view.LoginActivity;


@Subcomponent(modules = LoginActivityModule.class)
public interface LoginActivityComponent extends AndroidInjector<LoginActivity> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<LoginActivity> {}

}

File: app/src/main/java/mb00/android/codehub/ui/login/inject/LoginActivityComponent.kt
package mb00.android.codehub.ui.login.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.login.view.LoginActivity


@Subcomponent(modules = [
    LoginActivityModule::class
])
interface LoginActivityComponent : AndroidInjector<LoginActivity> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<LoginActivity>()

}
